sdcm/utils/common.py

```diff
@@ -2058,7 +2058,12 @@ def get_ami_tags(ami_id, region_name):
     if test_image.tags:
         return {i['Key']: i['Value'] for i in test_image.tags}
     else:
-        return {}
+        new_ec2_resource = get_scylla_images_ec2_client(region_name=region_name)
```

Comment:
```suggestion
        scylla_images_ec2_resource = get_scylla_images_ec2_client(region_name=region_name)
```

we probably should rename `get_scylla_images_ec2_client` to `get_scylla_images_ec2_resource`
not that we think about it, I wrongly named it client... which might be confusing a bit.

I would use the word new here, it's maybe fallback

we should also consider to reverse the order, first lookup in the images account
and if not found fallback to the original account.
so at the end we'll do one call going forward, and no two lookups going forward.

Reply:
renamed the function and reversed the order of the calls